src/DigitKeyboard/controller.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 
-import { EventDispatcher } from './eventDispatcher.js';
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from '../../node_modules/string-to-dom';
```

Comment:
Node modules by names, not by relative paths (although possible). So whenever you are using npm module, you load it with:
`import stringToDom from 'string-to-dom'`

Browserify will know how to resolve paths automatically.